config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
```

Comment:
bu dosyayi da anlamadim.

Reply:
Abi bunlar rails 6.1 ile geldi. rails app:update dediğin zaman geliyor.